src/Pages/Main/Main.js

```diff
@@ -1,13 +1,14 @@
 import React from 'react';
 import Footer from '../../components/Footer/Footer';
 import Header from '../../components/Header/Header';
+import Item from '../../components/Item/Item';
 
 function Main() {
   return (
     <div>
       <main className='main-content'>
         <Header />
-        <p>Main page</p>
+        <Item />
```

Comment:
The Design item wont be added directly to the main page but inside a DesignList component #57. At this point it it enough to create the component with unit test